dir.targets

```diff
@@ -61,5 +61,6 @@
   <PropertyGroup Condition="$(AssemblyOriginatorKeyFile.Contains('Open.snk'))">
     <StrongNameSig />
     <DelaySign>false</DelaySign>
-  </PropertyGroup> 
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
```

Comment:
It causes /publicsign+ to be passed to CSC which makes it do oss signing instead of real signing (real signing still does not work on *nix, and so without this Roslyn errors out saying signing is not supported on this platform).